students/jstrauss123/lesson03/mailroom.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+# function - send thank you
+def fun_sendty():
+    full_name = input("Please provide full name of donor: ")
+    while full_name == "list":
+        print(donor_list)
+        full_name = input("Please provide full name of donor: ")
+    if full_name in donor_list:
+        #print("full name entered is: ", full_name)
+        idx_num = donor_list.index(full_name)
+        #print("index is: ", idx_num)
+        #str1 = "donamt_" + str(idx_num)
+        #print("str1 value is: ", str1)
+        #print(" list value is equal to: ", str1[-1])
+        # if donor in list send thank you email
+        print("Dear {}, Thank you so much for your generous contribution".format(donor_list[idx_num]))
+    # if user not in donor_list add new donor and prompt for contribution amount
+    else: 
+        donor_list.append(full_name)
+        idx_num = donor_list.index(full_name)
+        new_donamt = input("Enter donation amount for new donor: ")
+        newamt = float(new_donamt)
+        print("Dear {}, Thank you so much for your generous contribution".format(donor_list[idx_num]))
+        # code here to create new contribution list for new donor
+        #donamt_<insert index here>.append(newamt)
+    #print("response is: ", response)    
+    #if response_ty == "list"
+    #    print(donor_list)
+    #if response_ty in donor_list:
+
+# function - create report
+def fun_report():
```

Comment:
if `fun_` indicates a "function" - no need; just `create_report` is fine

Reply:
OK, will redo function names, I was going with what I saw on instructor video. 
Please note mailroom.py was replaced by updated version mailroom1.py.

In the future if I make changes after a pull request, OR I am redoing a lesson based on instruction request what is the epected outcome? Should the redo be the same name as original, or lesson_v2 or?

Reply:
Yes you would update the same file as before please; multiple files for the same assignment would cause confusion.